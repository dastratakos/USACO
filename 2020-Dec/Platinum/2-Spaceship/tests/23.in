60 60 60
111011011001111110111100111111110110011111111111011010110111
111111111011111111010111111111010111100000110001111111110101
100010111101011001001101111111100000001101101101011010001101
011110010011110111110100110010100111100100101111111011011101
111111001101111101011011011111111111111010111101110100111111
100111000111000111111110110111111111010011010111011111101011
111111111010111001101011011011101100101110011111110111001101
010101100101111111000010010011110111111010110011110101011011
110111111111000011111010110101111011111111100111111110111101
010110101001111001011111001001000111001001011000001111101111
111001000100010111010111111101101100101000111111110101111111
011101011101101011100111111100101111100011101111100110111010
110110111000010101100100011111111101000101110111111111011000
111111101011011010100111110011011000011011011111011101101101
101111101101111110010011111111011111100111110101010011111100
011110110011101000011001111110101001011111010010111111010101
111101011110100110111011010011011111011011111110111111110001
010100111011101001010111010110011101111011000100111110100110
010110001111100100110111110011111001000010011011100111011101
011111110010110111011111101101101111101110011101101110101011
010000011010011110111011111010111001101111011110011101001110
101110111111111101010001011101101101111111101101110110011111
110101101010011111010111111111001111111111011101000000101111
011101101111111111101101101010111100001101011100111011010011
011110111001100111101011111011001111001011110101111110001101
111101111001101000111011101111110011101111011110111111101111
110101111011110110100111010011111100001001011111010000111011
100010101011101110001111111011000100110011111111111000010011
010011100101101110100111110110000111011110111111111111111010
010101100110110111110011111111001000101001111000101101101001
110111110111100011011111010101011101001110111011010010110111
010101000111111101111111001101101110111110111011011011101111
110110111011001011100011101100001110101000111110111011001110
111011001110011110101111110011100011111011111011011110010011
011111101010111011111011101011111100011011101111011110100111
011001100111011100011110110101011100111011110101010101110111
111010111111110101110001011000111111101110010100111000011100
100011001111010101110111001000010100111111000011010111111011
111011111100011110110101110001110111110000110011011000100111
010111001101110011000110111110111111110111011111010000011101
011111011000111101111111111011111010111001101001010011110110
101110101011010101101111111111001001111011001000111010110000
001101110001010111101100011111011011011111110110100001000111
111111011111001101000111111111111111111110011011101111111111
111110011111011110111101111101111111110100100110111100110101
101010111101001110110011010111111000011101111000111101111011
111101111101110101101011111110111110101111110111111101001010
110101011101010011100111100110111111010111100110110111111101
011000001010001110011101110011111110111011011111011101111011
100001111111101111100110101100110101111110010111110011111101
110001110111110111111000001001111100001111111011011101000011
101010101101101101100011111010111110001111111110111111111001
110010101010010011111111011001110111111101101000110000001111
101101001001111011010100111010101011011111110111101101100001
110111101101001110111011101111111101101011001111110101101111
010110011011111111001101111011101111110001101111111101110010
101111011011111010010110010111111110111100001110101001111111
111100010001001110010110111111111111000111010011111111110111
111110010110011101110001101101000011100100110111111011011110
111111101111100111101100110010111000101110101111111001111111
1 44 3 24
1 33 2 12
16 49 9 51
3 21 4 39
2 20 5 17
4 31 2 34
4 5 3 35
3 35 33 22
34 56 2 20
4 9 1 22
4 43 5 41
4 17 3 50
43 18 3 55
5 48 2 25
3 58 1 9
2 44 5 21
1 48 1 26
5 55 3 9
3 11 2 40
5 42 3 36
3 17 52 58
4 10 19 15
3 24 2 29
3 32 1 41
2 51 2 7
2 45 1 19
1 49 53 8
45 26 5 31
2 41 2 59
4 58 3 35
57 59 5 10
4 33 4 51
3 41 35 37
5 21 2 10
1 48 5 46
52 9 1 42
4 36 3 45
4 30 5 55
2 30 5 10
5 4 2 6
54 30 28 13
3 26 2 37
24 41 29 18
4 42 3 26
4 25 60 29
1 39 3 40
6 4 1 28
36 44 5 19
4 2 2 8
43 23 4 14
5 6 58 2
2 50 3 17
3 16 2 58
1 30 39 10
1 8 5 7
3 39 3 53
2 21 3 34
1 22 4 40
2 2 3 52
1 24 1 24
