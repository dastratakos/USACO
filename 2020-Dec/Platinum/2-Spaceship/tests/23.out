356596877
441566612
179333165
311294369
267656603
703411289
219226473
728052015
288908648
455385192
228836868
374644248
494514849
503767083
263233318
18854480
493737608
347629389
446732223
250050219
170404093
70017782
212342906
990768281
95703122
273603547
685643604
353814169
225807386
897490754
53035573
783508818
720853438
566702393
706587234
577907107
50085323
213886835
641161153
262483256
674897618
482413793
803307133
185021853
30931827
487720115
844854380
502886899
726860373
636787387
81506984
47845775
902058690
902450772
182127445
746479312
813893466
77977485
59941037
408622267
